jobs/cleanup.sh

```diff
@@ -15,6 +15,8 @@ if [ -z "$DEBUG_JOB" ] || [ "$DEBUG_JOB" != "yes" ]; then
     source /home/jenkins-slave/tools/keystonerc_admin
     nova delete $VMID
     /usr/local/src/cinder-ci/vlan_allocation.py -r $VMID
+
+    rm -f /home/jenkins-slave/runs/devstack_params.$ZUUL_UUID.$JOB_TYPE.txt
```

Comment:
Leaving this file there helps with nothing, even for debug, as no other job will pick it up.